src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/GeneralEmailService.java

```diff
@@ -55,7 +55,8 @@ private CaseDocument createNewCaseDocumentObject(GeneralEmailWrapper wrapper) {
             if (url != null) {
                 documentToReturn.setDocumentUrl(url);
             }
+            return documentToReturn;
         }
-        return documentToReturn;
+        return null;
```

Comment:
This again null return will create null pointer exception

Reply:
document is optional if not uploaded then it should be null